.github/workflows/publish-and-test.yml

```diff
@@ -138,13 +138,12 @@ jobs:
       - name: Build and push Docker image
         uses: docker/build-push-action@v5
         with:
-          platforms: linux/amd64,linux/arm64
```

Comment:
Hello @CalebSLane any reason youre dropping the multi platform build ??

Reply:
I dropped it because I don't know if it was contributing much, but was delaying the github workflows pretty dramatically. Was originally done to help the ozone team, but they had said they still couldn't run it on their mac hardware. If we have an indication that it is actually working/necessary, I'd be happy to have it integrated back in, but we should maybe look at leveraging parallel builds